crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -419,6 +420,9 @@ pub struct AuthorityEpochTables {
 
     /// Transactions that are being deferred until some future time
     deferred_transactions: DBMap<DeferralKey, Vec<VerifiedSequencedConsensusTransaction>>,
+
+    /// Records the round numbers for which we have written randomness.
+    randomness_rounds_written: DBMap<u64, ()>,
```

Comment:
this could probably be `DBMap<(), u64>` instead since it is effectively just a watermark, right? (only need to record highest round)